recipe/build-core.sh

```diff
@@ -16,17 +16,13 @@ if [[ "$target_platform" == osx* ]]; then
   sed -i"''" -e'/stdlib=libc/d' $CONDA_PREFIX/share/bazel_toolchain/CROSSTOOL.template
   source gen-bazel-toolchain
   cat >> .bazelrc <<EOF
-build --define CONDA_CC=${CC}
-build --define CONDA_CFLAGS="${CFLAGS}"
-build --define CONDA_AR=${AR}
-build --define CONDA_NM=${NM}
-build --define CONDA_RANLIB=${RANLIB}
-build --define CONDA_SDKROOT=${SDKROOT}
-# build --subcommands
 build --crosstool_top=//bazel_toolchain:toolchain
-build --cpu=${TARGET_CPU}
 build --platforms=//bazel_toolchain:target_platform
 build --host_platform=//bazel_toolchain:build_platform
+build --extra_toolchains=//bazel_toolchain:cc_cf_toolchain
+build --extra_toolchains=//bazel_toolchain:cc_cf_host_toolchain
+build --toolchain_resolution_debug
+build --define=with_cross_compiler_support=true
 build --experimental_ui_max_stdouterr_bytes=16000000
 build --local_ram_resources=HOST_RAM*.8 --local_cpu_resources=2
 EOF
```

Comment:
Started with something [more similar to Jax]( https://github.com/conda-forge/jaxlib-feedstock/blob/998de7823fdd57c612d4d5af444e7dcd40af9167/recipe/build.sh#L81-L92 ). Though some of those flags were redundant [with `bazel-toolchain`]( https://github.com/conda-forge/bazel-toolchain-feedstock/blob/804ad52abfabc595ab541f0f6f8c5f4106b5a79f/recipe/gen-bazel-toolchain#L5 ). This can probably be tweaked further